hpc/nways/nways_labs/nways_MD/English/C/jupyter_notebook/openacc/nways_openacc_opt.ipynb

```diff
@@ -312,7 +312,7 @@
    "outputs": [],
```

Comment:
In the Opt lab . I see w ehave explained the Occupancy. But i do not see a link to GPU TErminologies

Reply:
Also i beleive the terminology is wrong:
An OpenACC gang is a threadblock
A worker is a warp
An OpenACC vector is a CUDA thread

Can you check once meore. I know that it was earlier
An OpenACC gang is a thread block
A worker is a threads in y direction
An OpenACC vector is threads in x direction

Reply:
Looking at flow . My suggestion would to split the notebook into 2 parts. First which finishes till Tile clause.
The next Notebook can be called as " HArdware Level Insights with Compute" Because we are not optimizing anything. Just giving insight and how to use Compute. The terminologies are also complicated for abeginner which are not covered in the GPU TErminoloes section e.g. What is difference beween Eligible vs Issued Warps. What makes them issue that instructions.

Reply:
- Things such as difference between Eligible vs Issued Wraps were not added yet.
- Splitted the notebooks but the OpenMP one now looks smaller.
- merge conflict fixed
- Terminologies were linked but not in a repetitive way
- For the terminologies please checkout slides shared on slack